src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -90,11 +90,17 @@ public void addNewInstructor(String name, String email) {
         waitForPageToLoad();
     }
 
-    public void editInstructor(int instrNum, String name, String email, String role) {
+    public void editInstructor(int instrNum, String name, String email,
+            Boolean isDisplayedToStudents, String displayName, String role) {
         clickEditInstructorLink(instrNum);
 
         editInstructorName(instrNum, name);
-        editInstructorEmail(instrNum, email);
+        if (getEmailField(instrNum).getAttribute("readonly") == null) {
+            editInstructorEmail(instrNum, email);
+        }
+        if (editInstructorDisplayedToStudents(instrNum, isDisplayedToStudents)) {
```

Comment:
Fixed in https://github.com/TEAMMATES/teammates/pull/7559/commits/fbb08257197813ea23abb7bc1d9c90004badf3e4